lib/solidity/selectors/index.js

```diff
@@ -81,7 +81,7 @@ let solidity = createSelectorTree({
           // context types. TODO
           sourceMap = "";
           for (var i = 0; i < instructions.length; i++) {
-            sourceMap += i + ":" + i + ":1:o;";
+            sourceMap += i + ":" + i + ":1:-1;";
           }
```

Comment:
What is file index -1? Is it for solc injected instructions?

Reply:
Yep: http://solidity.readthedocs.io/en/v0.4.22/miscellaneous.html#source-mappings